content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -588,3 +597,299 @@ As part of the Peruvian localization, besides creating credit notes from an exis
+use this feature.
+
+The use of the guía de remisión electrónica electronic document is mandatory and required by |SUNAT|
+for taxpayers who need to transfer their products, with the exception of those under the *Single
+Simplified Regime* (régimen único simplificado or RUS).
+
+Delivery guide types
+~~~~~~~~~~~~~~~~~~~~
+
+Sender
+******
+
+The *Sender* delivery guide type is issued when a sale is made, a service is rendered (including
+processing), goods are assigned for use, or goods are transferred between premises of the same
+company and others.
+
+This delivery guide is issued by the owner of the goods (i.e., the sender) at the beginning of the
+shipment. The sender delivery guide is supported in Odoo.
+
+.. seealso::
+   `SUNAT guía de remisión <https://www.gob.pe/7899-guia-de-remision>`_
+
+Carrier
+*******
+
+The *Carrier* delivery guide type justifies the transportation service performed by the driver (or
+carrier).
+
+This delivery guide is issued by the carrier and must be issued to each shipper when the shipment
+goes through public transport.
+
+.. important::
+   The carrier delivery guide is **not** supported in Odoo.
+
+.. seealso::
+   `SUNAT guía de remisión transportista
+   <https://tefacturo.pe/blog/sunat/guia-de-remision-electronica/guia-de-remision-transportista/>`_
+
+Transportation types
+~~~~~~~~~~~~~~~~~~~~
+
+Private
+*******
+
+The *Private* transportation type option is used when the owner carries out the transfer of goods
+using their own vehicles. In this case, a sender's delivery guide must be issued.
+
+Public
+******
+
+The *Public* transportation type option is used when an external carrier moves the goods. In
+this case, two delivery guides must be issued: the sender's delivery guide and the carrier's
+delivery guide.
+
+Changes in version 2.0
+~~~~~~~~~~~~~~~~~~~~~~
+
+The following changes have been made in version 2.0 of the |GRE| delivery guide.
+
+Direct submission to SUNAT
+**************************
+
+The creation of the |GRE| delivery guide in Odoo **must** be sent directly to the |SUNAT|,
+regardless of the electronic document provider: IAP, Digiflow or |SUNAT|.
+
+New required information
+************************
+
+Version 2.0 requires additional information on the general configuration, vehicles, contacts, and
+products. In the general configurations, it is now necessary to add new credentials that are
+acquired in the |SUNAT| portal.
+
+Flow
+****
+
+.. image:: peru/peru-flow-remision.png
+   :align: center
+   :alt: Flow for Guia de Remision emission.
+
+#. Generate sales order
+#. Validate delivery order
+#. Generate waybill
+#. Perform validation or rejection
+#. Send CDR
+#. Receive validated CDR on outbound order
+#. Carrier travels with printed |GRE|
+#. Deliver goods
+
+Cancellations
+*************
+
+Currently, the |SUNAT| no longer uses the term "Anula", but now uses the term "Dar de baja" for
```

Comment:
Is it currently in version 16?  will this PR also be implemented for the next versions? I find using currently misleading. We need to keep this factual and not refer to versions, since the users can select the version, they know what you're referring to

Reply:
Hello @chiaraprattico @samueljlieber the word "currently" is not referring to Odoo versions, but a change in SUNAT's terminology. It is mostly related to the term that its used now and clarifies why in Odoo will use the "current" term which is "Dar de baja"

Reply:
Hello @samueljlieber and @jcs-odoo I agree we can delete the first note regarding the past and present terms for cancellations.
Nevertheless I recommend adding an "Important" block at the end of the Cancellations section to clarify (I will add this in the correct section).
On the other hand the information regarding when it is possible for the sender/carrier to cancel the document is the policy to follow when cancelling in the SUNAT portal